packages/react-select/src/components/Select/useSelectStyles.ts

```diff
@@ -0,0 +1,109 @@
+import { makeStyles, mergeClasses } from '@fluentui/react-make-styles';
+import type { SelectState } from './Select.types';
+import type { Theme } from '@fluentui/react-theme';
+
+const contentSizes = {
+  // TODO: borrowed this from Input, should be in the theme somewhere?
+  body1: (theme: Theme) => ({
+    fontSize: theme.fontSizeBase300,
+    lineHeight: theme.lineHeightBase300,
+  }),
+  caption1: (theme: Theme) => ({
+    fontSize: theme.fontSizeBase200,
+    lineHeight: theme.lineHeightBase200,
+  }),
+  // eslint-disable-next-line @typescript-eslint/naming-convention
+  400: (theme: Theme) => ({
+    fontSize: theme.fontSizeBase400,
+    lineHeight: theme.lineHeightBase400,
+  }),
+};
+// TODO: borrowed this from Input (Select has the same size in design comps)
+// Should be in the theme somewhere?
+const fieldHeights = {
+  small: '24px',
+  medium: '32px',
+  large: '40px',
+};
+const borderRadius = (theme: Theme) => theme.borderRadiusMedium;
+
+// TODO: borrowed this from Input, should be in the theme somewhere?
+// Form fields with text values share the same size/appearance variants/font sizes
+const backgroundColors = {
+  filledLighter: (theme: Theme) => theme.colorNeutralBackground1,
+  filledDarker: (theme: Theme) => theme.colorNeutralBackground3,
+  transparent: (theme: Theme) => theme.colorTransparentBackground,
+};
+
+const useRootStyles = makeStyles({
+  base: theme => ({
+    fontFamily: theme.fontFamilyBase,
+    boxSizing: 'border-box',
+    '*, *:before, *:after': {
+      boxSizing: 'border-box',
+    },
+  }),
+});
+
+const useSelectElementStyles = makeStyles({
+  base: theme => ({
+    appearance: 'none',
+    color: theme.colorNeutralForeground1,
+    borderRadius: borderRadius,
+
+    ':focus-visible': {
+      outline: '2px solid transparent',
+    },
```

Comment:
can we reuse the default focus outline styles as other components ? We can add a transparent outline there (it's not there currently), but I recently cleared up with Design that we can use it for windows HC

Reply:
Select and Input actually don't have the same focus styles as other components, so the default outline styles wouldn't apply. I've removed all styles from this PR though, in favor of the styles one :)